src/wallet/wallet2.cpp

```diff
@@ -1484,9 +1526,21 @@ void wallet2::process_outgoing(const crypto::hash &txid, const cryptonote::trans
     entry.first->second.m_subaddr_account = subaddr_account;
     entry.first->second.m_subaddr_indices = subaddr_indices;
   }
+  else
```

Comment:
This `else` causes the block below to be _not_ executed when the insertion took place. I believe this is unwanted.

Reply:
If it's not inserted, it would be because it was already seeded from the unconfirmed transaction structure, which had the ring already.
That said, I suppose setting it all the time can't hurt either as it should never be different, so I'll do that as it's more robust.

Reply:
Yes, I found it problematic when I did `rescan_bc`.